docker-hub/registry-access-management.md

```diff
@@ -0,0 +1,113 @@
+---
+description: Registry Access Management
+keywords: registry, access, managment
+title: Registry Access Management
+---
+
+Registry Access Management is a feature available to organizations with a Docker Business subscription. This feature lets organization owners manage the registries that their developers can access while using Docker Desktop.  When using this feature, organization owners can ensure that their developers can only access their trusted registries, such as a secure private registry on Artifactory, thereby reducing the security risks that can occur when developers interact with public registries.
+
+> **Note**
+>
+> Registry Access Management is currently offered as a Tech Preview to a closed group of Docker Business customers.
+{: .important}
+
+## Configure Registry Access Management permissions
+
+To configure Registry Access Management permissions, perform the following steps:
+
+1. Sign into your [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} account as an organization owner.
+2. Select an organization, navigate to the **Settings** tab on the **Organizations** page and click **Org Permissions**.
+3. Enable Registry Access Management to set the permissions for your registry.
+
+> **Note**
+>
+> When enabled, the Docker Hub registry is set by default, however you can also restrict this registry for your developers.
+
+4. Click **Add** and enter your registry details in the applicable fields, and click **Create** to add the registry to your list.
```

Comment:
On the preview page I am seeing items 4 and 5 incorrectly numbered as items 1 and 2. https://deploy-preview-14300--docsdocker.netlify.app/docker-hub/registry-access-management/

Reply:
@jerae-duffin Could you fix this. The note after 1 and 2 needs indenting

Reply:
Updated